deploy/helm/README.md

```diff
@@ -0,0 +1,40 @@
+# runtime-class-manager
+
+runtime-class-manager is a Kubernetes operator that manages installation of Wasm shims onto nodes and related Runtimeclasses via [Shim custom resources](../../config/crd/bases/runtime.kwasm.sh_shims.yaml).
+
+## Prerequisites
+
+- [Kubernetes v1.20+](https://kubernetes.io/docs/setup/)
+- [Helm v3](https://helm.sh/docs/intro/install/)
+
+## Installing the chart
+
+The following installs the runtime-class-manager chart with the release name `rcm`:
+
+```shell
+helm upgrade --install \
+  --namespace rcm \
+  --create-namespace \
+  --wait \
+  rcm .
+```
+
+## Post-installation
```

Comment:
Should this be added to a NOTES.txt?

Reply:
Good call.  What do you think of https://github.com/spinkube/runtime-class-manager/pull/204/commits/41de076b44016e0f9c1554eea80e1359b30dc436?